bricks/test_optimizer/CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+## 🦄 Welcome
+
+Hello! If you’re interested in contributing to Very Good CLI's test optimizer you’re reading the right document. First of all, thank you for showing interest in contributing! Before you do so it is important to carefully read this [other file](../../CONTRIBUTING.md) and [this file](CONTRIBUTING.md) before contributing.
```

Comment:
This has a self referencing infinite loop :P 

I dont think we need to link to the same document when we talk about `this file` right?